roboprop_client/views.py

```diff
@@ -157,6 +160,39 @@ def _get_suggested_tags(thumbnails):
     return tags, categories, colors
 
 
+def _get_blendkit_model_details(result):
+    return {
```

Comment:
I may have missed it, but there is no existing relevamt metadata for us to get, like a description?

Reply:
There is yes, and it will come after the sdf conversion (next PR) and upload, as blenderproc then provides us with some very good metadata (descriptions, tags, categories). So yes, but after the fact if that sounds ok?

But of course descriptions of models (for both fuel and blendkit) before "selecting" should happen. I will make a separate issue for that

Reply:
Sure!